packages/mediapipe-task-text/lib/src/io/tasks/text_classification/text_classifier_result.dart

```diff
@@ -45,9 +50,9 @@ class TextClassifierResult extends BaseTextClassifierResult with TaskResult {
 
   @override
   void dispose() {
-    if (_pointer != null) {
+    if (_pointer != null && !_isClosed) {
```

Comment:
Why would dispose be called multiple times?

Reply:
By error, I suppose? I had assumed that our pattern would be to make subsequent disposal calls a no-op, but your question got me looking and I now see that `ChangeNotifier.dispose` emits debug-only warnings about repeated calls. I can apply that here, as well.